### DISCUSSION TITLE
Is there a way to have AdGuardHome to Remove UTM tags from URLs?

### DISCUSSION THREAD

**Discussion by muhepd** [AdguardTeam/AdGuardHome#4117] [Q&A]
Hi,

Like the title says, Is there a way to have AdGuardHome avoiding the tracking or completely remove the UTM tags from URLs when resolving it?

Like these: utm_source=facebook.com&utm_medium=social&utm_campaign=lsfb

Thanks.

**Comment by DandelionSprout** [comment]
Sadly no.
